connections/api/rpc_views.py

```diff
@@ -0,0 +1,95 @@
+import logging
+from django_etuovi.etuovi import create_xml_file
+from django_oikotie.oikotie import create_apartments, create_housing_companies
+from elasticsearch_dsl import Search
+from rest_framework import permissions, status
+from rest_framework.decorators import action, permission_classes
+from rest_framework.response import Response
+from rest_framework.viewsets import ViewSet
+
+from connections.etuovi.etuovi_mapper import map_apartment_to_item
+from connections.oikotie.oikotie_mapper import (
+    map_oikotie_apartment,
+    map_oikotie_housing_company,
+)
+from connections.utils import create_elastic_connection
+
+_logger = logging.getLogger(__name__)
+
+
+class ConnectionsRPC(ViewSet):
+    """
+    An RPC class for calling special prosedures via api.
+    """
+
+    create_elastic_connection()
+
+    @permission_classes((permissions.IsAuthenticatedOrReadOnly,))
```

Comment:
Was this for test purposes? If so, recommend that we don't leave tests to the code that we don't forget to remove those later.